### PR TITLE
DEV-15002: Open up different ports in EMR cluster depending on whether we are on pre-6x or post-6x

### PR DESCRIPTION
**Pull request by msoni-tamr**

### CODE REVIEW

File: modules/aws-emr-ports/variables.tf [hunk 1]
```diff
@@ -17,7 +17,13 @@ variable "master_ports_spark" {
   ]
 }
 
-variable "master_ports_hbase" {
+variable "is_pre_6x" {
+  type        = bool
+  description = "Ports used by AWS EMR"
```
Comment (souza-dan) [inline modules/aws-emr-ports/variables.tf:22]: Description seems a bit off.

File: modules/aws-emr-ports/output.tf [hunk 2]
```diff
@@ -27,7 +29,7 @@ output "ingress_core_ports" {
     setunion(
       local.output_hbase_core,
       local.output_spark_core,
-      var.core_ports_emr,
+      var.is_pre_6x ? var.core_ports_emr_pre_6x : var.core_ports_emr_6x,
```
Comment (souza-dan) [inline modules/aws-emr-ports/output.tf:32]: Should we handle this in the locals instead similar to the master ports?

**Review by souza-dan — COMMENTED** [review]
structure looks good! Make sure to update the changelog, version, and run `make terraform/docs`.

**Review by souza-dan — COMMENTED** [review]
You'll want to update the version file too:
https://github.com/msoni-tamr/terraform-aws-emr/blob/master/VERSION

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+* Adds variables:
+  * `is_pre_6x`
+  * `master_ports_hbase_common`
+  * `master_ports_hbase_pre_6x`
+  * `master_ports_hbase_6x`
+  * `core_ports_emr_pre_6x`
+  * `core_ports_emr_6x`
+* Removes variables:
+  * `master_ports_hbase`
+  * `core_ports_emr`
```
[the remark this answers is not in the document]
Comment (msoni-tamr) [reply CHANGELOG.md:14]: The variables aren't really deprecated - more like replaced, so I'll go ahead and set it as a major release.

[the remark this answers is not in the document]
Comment (souza-dan) [reply CHANGELOG.md:14]: Right, we could have opted to deprecate them though, like what we did for when we additional_tags became tags
https://github.com/Datatamer/terraform-aws-emr/blob/master/variables.tf#L49-L53

I think just removing them is the right move though. I don't think anyone is actually using them, but we want to be firm about what we call major / minor / patch changes.

**Review by souza-dan — APPROVED** [review]
LGTM

**Pull request merged**